Mathlib/Data/Multiset/Antidiagonal.lean

```diff
@@ -37,80 +37,91 @@ theorem antidiagonal_coe (l : List α) : @antidiagonal α l = revzip (powersetAu
 
 @[simp]
 theorem antidiagonal_coe' (l : List α) : @antidiagonal α l = revzip (powersetAux' l) :=
-  Quot.sound revzip_powerset_aux_perm_aux'
+  Quot.sound revzip_powersetAux_perm_aux'
 #align multiset.antidiagonal_coe' Multiset.antidiagonal_coe'
 
+/- Porting note: Mathport seems to be replacing `quotient.induction_on s $` with
+`(Quotient.induction_on s)`, when it should be `Quotient.inductionOn s <|`. I'm more concerned
+about the parentheses being used instead of `<|` than the typo. -/
+/- Porting note: `simp` seemed to be applying `antidiagonal_coe'` instead of `antidiagonal_coe`
+in what used to be `simp [antidiagonal_coe]`. -/
 /-- A pair `(t₁, t₂)` of multisets is contained in `antidiagonal s`
     if and only if `t₁ + t₂ = s`. -/
 @[simp]
 theorem mem_antidiagonal {s : Multiset α} {x : Multiset α × Multiset α} :
     x ∈ antidiagonal s ↔ x.1 + x.2 = s :=
-  (Quotient.induction_on s) fun l => by
-    simp [antidiagonal_coe]; refine' ⟨fun h => revzip_powerset_aux h, fun h => _⟩
+  Quotient.inductionOn s <| fun l ↦ by
+    dsimp [antidiagonal_coe]; refine' ⟨fun h => revzip_powersetAux h, fun h ↦ _⟩
     haveI := Classical.decEq α
-    simp [revzip_powerset_aux_lemma l revzip_powerset_aux, h.symm]
+    simp [revzip_powersetAux_lemma l revzip_powersetAux, h.symm]
```

Comment:
Please replace the `simp`s with the output of `simp?`. That should have already been done in mathlib3.

Reply:
For every `simp` in the file or just those?

Reply:
Just these two. But in general, we should try to avoid non-terminal simps.
See https://leanprover-community.github.io/glossary.html#non-terminal-simp